app/Activity.php

```diff
@@ -14,6 +14,6 @@ class Activity extends Model
      * @var array
      */
     protected $fillable = [
-        'patrol_start_date', 'patrol_end_date', 'start_time', 'end_time', 'details', 'priorities','patrol_area', 'type', 'user_id'
+        'patrol_start_date', 'patrol_end_date', 'start_time', 'end_time', 'total_time', 'details', 'priorities','patrol_area', 'flag', 'type', 'user_id'
```

Comment:
Don't make a fillable for 'total_time', instead, pull it from calculations, this will become an issue when editing patrol logs in the future.